controllers/consoleplugin/config/static-frontend-config.yaml

```
@@ -803,3 +803,184 @@ filters:
+    lokiLabel: true
+  - name: DstK8S_OwnerType
+    type: string
+    description: Kind of the destination owner, such as Deployment, StatefulSet, etc.
+  - name: DstK8S_Namespace
+    type: string
+    description: Destination namespace
+    lokiLabel: true
+  - name: DstAddr
+    type: string
+    description: Destination IP address (ipv4 or ipv6)
+  - name: DstPort
+    type: number
+    description: Destination port
+  - name: DstMac
+    type: string
+    description: Destination MAC address
+  - name: DstK8S_HostIP
+    type: string
+    description: Destination node IP
+  - name: DstK8S_HostName
+    type: string
+    description: Destination node name
+  - name: DstK8S_Zone
+    type: string
+    description: Destination availability zone
+    lokiLabel: true
+  - name: K8S_FlowLayer
+    type: string
+    description: "Flow layer: 'app' or 'infra'"
+  - name: Proto
+    type: number
+    description: L4 protocol
+  - name: Dscp
+    type: number
+    description: Differentiated Services Code Point (DSCP) value
+  - name: IcmpType
+    type: number
+    description: ICMP type
+  - name: IcmpCode
+    type: number
+    description: ICMP code
+  - name: Duplicate
+    type: boolean
+    description: Indicates if this flow was also captured from another interface on the same host
+    lokiLabel: true
+  - name: FlowDirection
+    type: number
+    description: |
+      Flow direction from the node observation point. Can be one of: +
+      - 0: Ingress (incoming traffic, from the node observation point) +
+      - 1: Egress (outgoing traffic, from the node observation point) +
+      - 2: Inner (with the same source and destination node)
+    lokiLabel: true
+  - name: IfDirection
+    type: number
+    description: |
+      Flow direction from the network interface observation point. Can be one of: +
+      - 0: Ingress (interface incoming traffic) +
+      - 1: Egress (interface outgoing traffic)
+  - name: Interface
+    type: string
+    description: Network interface
+  - name: Flags
+    type: number
+    description: |
+      Logical OR combination of unique TCP flags comprised in the flow, as per RFC-9293, with additional custom flags to represent the following per-packet combinations: +
+      - SYN+ACK (0x100) +
+      - FIN+ACK (0x200) +
+      - RST+ACK (0x400)
+  - name: Bytes
+    type: number
+    description: Number of bytes
+  - name: Packets
+    type: number
+    description: Number of packets
+  - name: PktDropBytes
+    type: number
+    description: Number of bytes dropped by the kernel
+  - name: PktDropPackets
+    type: number
+    description: Number of packets dropped by the kernel
+  - name: PktDropLatestState
+    type: string
+    description: TCP state on last dropped packet
+    filter: pkt_drop_state # couldn't guess from config
+  - name: PktDropLatestDropCause
+    type: string
+    description: Latest drop cause
+    filter: pkt_drop_cause # couldn't guess from config
```

```suggestion
  - name: PktDropLatestState
    type: string
    description: TCP state on last dropped packet
  - name: PktDropLatestDropCause
    type: string
    description: Latest drop cause
```

We keep filter definition in the columns right ?